sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java

```diff
@@ -744,26 +824,22 @@ private Mono<CosmosBatchResponse> executeBatchRequest(PartitionKeyRangeServerBat
     private void completeAllSinks() {
         logger.info("Closing all sinks, Context: {}", this.operationContextText);
 
-        executorService.shutdown();
         logger.debug("Executor service shut down, Context: {}", this.operationContextText);
-        mainSink.tryEmitComplete();
-        logger.debug("Main sink completed, Context: {}", this.operationContextText);
-        groupSinks.forEach(FluxSink::complete);
-        logger.debug("All group sinks completed, Context: {}", this.operationContextText);
-
-        try {
-            this.executorService.shutdown();
-            logger.debug("Shutting down the executor service");
-        } catch (Exception e) {
-            logger.warn("Failed to shut down the executor service", e);
+        Sinks.EmitResult completeEmitResult = mainSink.tryEmitComplete();
```

Comment:
confirmation: before the change, we also complete all groupSinks in this method, is it intentionally remove the groupSink complete signal? 

(but i think it is fine to remove it as the during the main sink completion signal, we kind try to complete all group sinks as well)